src/api/storage-buckets.tsx

```diff
@@ -67,6 +67,24 @@ export const createStorageBucket = async (
     });
 };
 
+export const updateStorageBucket = async (
+  bucket: LxdStorageBucket,
+  pool: string,
+  project: string,
+): Promise<LxdOperationResponse> => {
```

Comment:
```suggestion
): Promise<void> => {
```
There is not an operation in the api response. I think we can simplify this to void, as we don't need the response upstream.